.github/workflows/release.yaml

```diff
@@ -1,375 +0,0 @@
-name: Release
-on:
-  workflow_dispatch:
-    inputs:
-      version:
-        type: string
-        description: The version you intend to release without leading 'v' (eg x.y.z and not vx.y.z)
-  pull_request:
-    types: [closed]
-  push:
-    branches:
-      - "release/**"
-      - "next/**"
-    tags:
-      - v*
-  release:
-    types: [created]
-
-env:
-  VERSION: ${{ github.event.inputs.version }}
-  APP_ID: 251311
-
-jobs:
-  prepare:
-    runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.version != '' }}
-    steps:
-      - name: Generate token
-        id: generate_token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
-        with:
-          app_id: ${{env.APP_ID}}
-          private_key: ${{ secrets.TOKEN_EXCHANGE_GH_APP_PRIVATE_KEY }}
-          permissions: >-
-            {"contents": "write", "pull_requests": "write"}
-      - name: Checkout repository code
-        uses: actions/checkout@v4
-        with:
-          token: ${{ steps.generate_token.outputs.token }}
-      - uses: actions/setup-go@v4
-        with:
-          go-version: "1.21.x"
-      - name: Install Buf
-        run: make installbuf
```

Comment:
Do we _really_ need to install buf here? I suppose it checks whether it builds, but regular ci.yaml should cover that.